lib/api/services/wallet_service.dart

```diff
@@ -139,7 +150,7 @@ class WalletService extends ApiService {
           throw Exception(balanceResponse.body);
         }
       } else {
-        throw Exception(blockchainResponse.body);
+        throw Exception("ERROR: Unable to get blockchain info");
```

Comment:
Why not show the actual error to the user?

Reply:
The error would have been shown before execution gets here. The function that should throw the error is in it own try-catch block already

Reply:
Updated